src/components/common/YourCompany/YourCompany.vue

```diff
@@ -283,7 +283,7 @@
     <v-divider class="mx-4 my-1" />
 
     <!-- Business Contact Information -->
-    <div id="contact-info" class="section-container" :class="{'invalid-section': invalidContactSection}">
+    <div id="contact-info-section" class="section-container" :class="{'invalid-section': invalidContactSection}">
```

Comment:
The previous id conflicted with the id in the component. This is both a HTML no-no and it didn't let me target the correct div to fix the left border radius (CSS below).

Before:
![image](https://user-images.githubusercontent.com/10002889/173142668-45f21c0b-e2f0-4c94-b609-944d7b618d74.png)

After:
![image](https://user-images.githubusercontent.com/10002889/173142726-d65d9eff-7aeb-42e5-a8d3-b641a403c507.png)